submissions/arinolaOsagie/HTML-FORMS/contact-form.html

```diff
@@ -6,7 +6,7 @@
 </head>
 <body>
     <nav>
-        <a href="http://localhost:63342/dufuna-html/ProductPage.html?_ijt=20eggs136c3k6r8671nrtog35b">Product</a> <a href="http://localhost:63342/dufuna-html/HTML%20FORM.html?_ijt=6p04hq2tlbksa86k5jsajp3b12">Contact Form</a>
+        <a href="productpagehtml.html">Product</a> <a href="contactpage.html">Contact Form</a>
```

Comment:
I am very sure these links don't redirect when you click them. You should have contact-form.html and not contactpage.html i.e the name of the file. For the product page also, ensure you are using the exact name of the file. 

The links should be in both the product page and contact-form files